src/test/java/com/olgaboiar/mint/ServerTest.java

```diff
@@ -3,60 +3,44 @@
 import com.olgaboiar.mint.loggers.FileLogger;
 import com.olgaboiar.mint.loggers.FileLoggerTest;
 import com.olgaboiar.mint.loggers.ILogger;
+import com.olgaboiar.mint.loggers.MockFileLogger;
 import org.junit.jupiter.api.*;
 
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.Socket;
+import java.nio.charset.StandardCharsets;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class ServerTest {
-    Server testServer;
-    String host = "localhost";
-    int port = 5000;
-    ILogger logger = new FileLogger("testLogs.txt");
+    ILogger logger = new MockFileLogger();
 
     @BeforeAll
     public void setUp () throws Exception {
-        testServer = new Server(host, port, logger);
+        MockServerConnection serverSocket = new MockServerConnection();
+        Server testServer = new Server(serverSocket, logger);
         testServer.start();
     }
 
-    @AfterAll
-    public void tearDown() throws IOException {
-        testServer.stop();
-    }
-
-    @Test
-    void testServerIsReachableAfterStarting () throws Exception {
-        InetAddress server = InetAddress.getByName("localhost");
-        boolean reachable = server.isReachable(7000);
-        assertTrue(reachable);
-    }
-
 //    @Test
-//    void testServerAcceptsClientConnection () throws Exception {
-//        Socket testClient = new Socket();
-//        testClient.connect(new InetSocketAddress("localhost", 5000));
-//        testServer.acceptClientConnection();
-//        boolean connected = testClient.isConnected();
-//        assertTrue(connected);
-//        testClient.close();
+//    void testServerAnswersGetRequest () throws Exception {
+//        String requestLine = "GET /simple_get HTTP/1.1";
+//        InputStream stream = new ByteArrayInputStream((requestLine).getBytes(StandardCharsets.UTF_8));
+//        System.setIn(stream);
+//
+//        String actual = new StringWriter().toString();
+//        String expected = String.join("\r\n", new String[]{
+//                "HTTP/1.1 200 OK",
+//                "Content-Length: 0",
+//                "",
+//                "",
+//                ""
+//        });
+//
+//        assertEquals(expected, actual);
```

Comment:
Remove commented out code.

Reply:
removed in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/4713bbd8fc081a7764a438b7c5a2c62956bad05f